hathor/manager.py

```diff
@@ -221,6 +223,14 @@ def __init__(self, reactor: Reactor, peer_id: Optional[PeerId] = None, network:
         else:
             self.capabilities = DEFAULT_CAPABILITIES
 
+        # This is included in some logs to provide more context
+        self.environment_info = environment_info
+
+        # Task that will count the total sync time
+        self.lc_check_sync_state = LoopingCall(self.check_sync_state)
+        self.lc_check_sync_state.clock = self.reactor
+        self.lc_check_sync_state_interval = 30  # seconds
```

Comment:
Move to a constant? It's ok to be a constant in the class.

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/490/commits/a273bdd4943e18022ce122159dcb88c3e7781903

Reply:
Thanks!